pkg/controller.v1/tensorflow/tfjob_controller.go

```diff
@@ -133,6 +133,9 @@ type TFJobReconciler struct {
 //+kubebuilder:rbac:groups=kubeflow.org,resources=tfjobs/finalizers,verbs=update
 //+kubebuilder:rbac:groups="",resources=pods,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups="",resources=services,verbs=get;list;watch;create;delete
+//+kubebuilder:rbac:groups=scheduling.volcano.sh,resources=podgroups,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=scheduling.sigs.k8s.io,resources=podgroups,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Can we add these markers to all controllers to clarify the needed roles in each controller?

Reply:
Done